chia/plot_sync/receiver.py

```diff
@@ -329,6 +336,9 @@ async def _sync_done(self, data: PlotSyncDone) -> None:
         self._keys_missing = self._current_sync.delta.keys_missing.additions.copy()
         self._duplicates = self._current_sync.delta.duplicates.additions.copy()
         self._total_plot_size = sum(plot.file_size for plot in self._plots.values())
+        self._total_effective_plot_size = sum(
+            UI_ACTUAL_SPACE_CONSTANT_FACTOR * int(_expected_plot_size(plot.size)) for plot in self._plots.values()
```

Comment:
how come `_expected_plot_size()` doesn't return the effective size? is "expected" different from "effective"?
I would expect the effective size depends on the compression level, does it not?

Reply:
`_expected_plot_size()` is a part of concensus, meaning it is used to calculate `require_iter` and it shouldn't depend on compression level. The difference between actual plot file size and effective plot size (roughly 101GiB) is what user earns by compression.